src/mdns/mdns_avahi.cpp

```diff
@@ -1170,10 +1172,15 @@ void PublisherAvahi::HostSubscription::HandleResolveResult(AvahiRecordBrowser *
     bool       resolved   = false;
     int        avahiError = AVAHI_OK;
 
+    otbrLogResultCode(
+        aEvent == AVAHI_BROWSER_FAILURE,
```

Comment:
```suggestion
        aEvent == AVAHI_BROWSER_FAILURE ? OTBR_ERROR_MDNS : OTBR_ERROR_NONE
```

Reply:
The purpose of `otbrLogResultCode` is to handle non-otbr errors, not otbr errors.